tests/test_compile_shaders.py

```
@@ -548,4 +591,1705 @@
+        assert args.max_warnings == 100
+        assert args.optimization == "2"
+        assert args.debug_defines_set == {"DEBUG", "VERBOSE"}
+
+
+def test_parse_arguments_missing_config():
+    """Test parse_arguments with missing config file argument."""
+    test_argv = ["prog", "--debug"]
+    with patch.object(sys, "argv", test_argv):
+        try:
+            parse_arguments(default_jobs=4)
+            assert False, "Should have raised SystemExit"
```

Ruff (0.11.9)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)